src/index.js

```diff
@@ -51,14 +80,17 @@ app.use((req, res, next) => {
 // Route definitions (add more here!)
 const example = require("./routes/example");
 app.use("/api/example", example);
+const account = require("./routes/auth");
+app.use("/api", account);
```

Comment:
```suggestion
app.use("/api/auth", account);
```

To ensure that routes in different files do not collide for sure (if another file used /api and defined the /login path for some reason there would be unexpected collisions that would be annoying to detect and whatnot)